zone/command.cpp

```diff
@@ -549,25 +416,51 @@ int command_add(std::string command_name, std::string description, uint8 admin,
 	}
 
 	auto c = new CommandRecord;
-	c->admin = admin;
-	c->description = description;
-	c->function = function;
 
-	commands_map[command_name] = admin;
+	c->admin       = admin;
+	c->description = description;
+	c->function    = function;
 
-	commandlist[command_name] = c;
+	commands_map[command_name]   = admin;
+	commandlist[command_name]    = c;
 	commandaliases[command_name] = command_name;
+
 	command_delete_list.push_back(c);
 	command_count++;
 
 	return 0;
 }
 
-uint8 GetCommandStatus(Client *c, std::string command_name) {
-	auto command_status = commands_map[command_name];
-	return command_status;
+uint8 GetCommandStatus(std::string command_name)
+{
+	return commands_map[command_name];
 }
 
+std::vector<CommandSubsettingsRepository::CommandSubsettings> GetCommandSubsettingsByCommand(std::string command)
+{
+	std::vector<CommandSubsettingsRepository::CommandSubsettings> v;
```

Comment:
We have `command_subsettings` in memory, we should use it instead of hitting the DB each command invocation

![image](https://github.com/EQEmu/Server/assets/3319450/87c7ee0a-fdcb-4194-8a92-8cab8ac24600)